### ISSUE TITLE
Inserting image does not move focus out of the toolbar 

### ISSUE THREAD

**Issue by mateuszzagorski** [ckeditor/ckeditor5#12229]
## 📝 Provide detailed reproduction steps (if any)

After using the insert image, the focus stays on the toolbar button:  

https://user-images.githubusercontent.com/11699569/183067079-b926fe8f-c25b-4595-b32f-dface3f32956.mov

### ✔️ Expected result

Focus should be moved onto the image.

### ❌ Actual result

Focus stays on the toolbar button.

## 📃 Other details

This is also happening on a non-collapsed toolbar.
All of the ui components in the `ckeditor5-image` should work correctly.

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by Dumluregn** [comment]
While working on it we should also ensure that https://github.com/ckeditor/ckeditor5/issues/12178#issuecomment-1205363543 is working as expected (it should be fixed with the same code but let's be sure).

**Comment by mlewand** [comment]
> While working on it we should also ensure that [#12178 (comment)](https://github.com/ckeditor/ckeditor5/issues/12178#issuecomment-1205363543) is working as expected (it should be fixed with the same code but let's be sure).

I wonder how is it possible that we never noticed it 😅 IMO it has a major impact on the UX of image feature with balloon toolbars.
